output_of_generator.js

```diff
@@ -0,0 +1,1045 @@
+function towire_tlv_payload_amt_to_forward(value)
```

Comment:
Hmm, how was output_of_generator.js produced?  We should make sure it can be reproduced (I add targetws to the toplevel makefile)

Reply:
```
file = open('../output_of_generator.js', 'w')
file.write('')
```
will add comments in the file as well..